internal/server/kong/ws/config/compat/compatibility_table.go

```diff
@@ -75,6 +75,7 @@ var (
 		"connect_timeout",
 		"send_timeout",
 		"read_timeout",
+		"http_response_header_for_traceid",
```

Comment:
This is for 3.1 not 3.0; it will need to be defined in its own 3.1 version compatibility instance.